bfscripts.spec.rpkg

```diff
@@ -0,0 +1,56 @@
+%global debug_package %{nil}
+
+Summary: Useful scripts for managing Mellanox BlueField hardware.
+Name: mlxbf-bfscripts
+Version: 3.0.alpha3
+Release: 1%{?dist}
+License: BSD
+
+Source: {{{ git_dir_pack dir_name=bfscripts source_name=mlxbf-bfscripts.tar.gz }}}
+
+BuildRequires: python3-devel
+BuildRequires: /usr/bin/pathfix.py
```

Comment:
Is it duplicate with the 'python3-devel'?

Reply:
This is required by the python packaging guidlines if you're building for python3 (https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_dependencies)